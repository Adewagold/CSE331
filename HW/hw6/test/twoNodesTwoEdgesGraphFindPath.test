# Tests find minimum-cost path if two paths exist.

CreateGraph two_nodes_two_edges_graph
AddNode two_nodes_two_edges_graph node1
AddNode two_nodes_two_edges_graph node2
AddEdge two_nodes_two_edges_graph node1 node2 1
AddEdge two_nodes_two_edges_graph node1 node2 1.001

FindPath two_nodes_two_edges_graph node1 node2