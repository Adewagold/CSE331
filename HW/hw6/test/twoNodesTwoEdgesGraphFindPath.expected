# Tests find minimum-cost path if two paths exist.

created graph two_nodes_two_edges_graph
added node node1 to two_nodes_two_edges_graph
added node node2 to two_nodes_two_edges_graph
added edge 1.000 from node1 to node2 in two_nodes_two_edges_graph
added edge 1.001 from node1 to node2 in two_nodes_two_edges_graph

path from node1 to node2:
node1 to node2 with weight 1.000
total cost: 1.000